pkg/agent/api/endpoints_windows.go

```diff
@@ -0,0 +1,14 @@
+//go:build windows
+// +build windows
+
+package api
+
+import (
+	"net"
+
+	"github.com/spiffe/spire/pkg/common/peertracker"
+)
+
+func (e *Endpoints) createListener() (net.Listener, error) {
+	return nil, peertracker.ErrUnsupportedPlatform
```

Comment:
This is coming in with a follow-up PR, right?

Reply:
Yes, we have https://github.com/spiffe/spire/issues/2833 filed for this.